File: FireFlux2-main/Codes/ROS_Codes/backfire_ROS_plot.m
%This code plots the backfire rate of spread with the modified adjr0 coefficient 
x = [0.90677 0.93 0.96 0.98 0.995 0.998 0.9999 1];
%y = [0.016343623213698418 0.016343623213698418 0.016968406861031387 ...
%    0.01733921815889029 0.018134509124451997 0.017712621441470035 ...
%    0.017739540319040657 0.022056307586713404];

y = [0.015989734039574453 0.016798221788879665 0.01709021600282394 ...
    0.01720178385015444 0.01767115575900313 0.017712621441470035 ...
    0.017739540319040657 0.02106970673272338];
plot(x, y, '-o')
title('ROS Coefficients vs. Calculated ROS','fontweight','bold','fontsize',18);
xlabel('ROS Coefficients', 'fontweight','bold','fontsize',16);
ylabel('ROS (m/s)','fontweight','bold','fontsize',16);
legend('ROS');
grid on;

File: FireFlux2-main/Codes/ROS_Codes/Backfire_ROS_Plot.py
# -*- coding: utf-8 -*-
"""
Created on Wed Jun  1 16:59:50 2022

@author: Jeremy Benik
"""
# %% Importing necessary libraries
import matplotlib.pyplot as plt
import numpy as np
import pandas as pd

# %% Inputting the x and y data
x = [0.9067704, 0.93, 0.96, 0.98, 1]
y = [0.016343623213698418, 0.016343623213698418, 0.016968406861031387, 0.01733921815889029, 0.022056307586713404]

fig = plt.figure(figsize = (12, 12))
plt.plot(x, y, label = 'Rate of Spread Adjustment vs. Rate of Spread', color = 'red', marker = 'o')
plt.title('Backfire ROS vs. Adjustment Factor', fontsize = 18, fontweight = 'bold')
plt.xlabel('adjr0 Coefficient', fontsize = 12, fontweight = 'bold')
plt.ylabel('Backfire ROS (m/s)', fontsize = 12, fontweight = 'bold')
plt.xlim(x[0], 1)
plt.grid()
plt.show()